### ISSUE TITLE
Remove document should be more obvious

### ISSUE THREAD

**Issue by tuomas2** [AndBible/and-bible#1876]
Similarly to #1804 (document information button in download/choose document sreen), document removal is far from obvious for many users. Should we have button for it visible too? Or 3dot menu per item like in workspaces view?

**Comment by agrogers** [comment]
Yes, more obvious is good. Need to try and keep things consistent with
other places
